ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala

```diff
@@ -421,6 +424,45 @@ class CreateTopicProgramSpec extends AnyWordSpecLike with Matchers {
       } yield topic shouldBe defined).unsafeRunSync()
     }
 
+    "ingest updated metadata into the metadata topic - verify deprecated date if supplied is not overwritten" in {
```

Comment:
Should we also have test cases for when no deprecated date is supplied? To make sure it is overwritten? Or maybe this test case already does that and I am just not seeing it

Reply:
Added